CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## [0.29.0](https://github.com/eclipse/lemminx/milestone/45?closed=1) (November 21, 2024)
+
+### Enhancements
+
+- ICompletionParticipant#onXMLContent is not called for CDATA. See [#1694](https://github.com/eclipse-lemminx/lemminx/issues/1694).
+
+### Bug Fixes
```

Comment:
Is https://github.com/eclipse-lemminx/lemminx/commit/d3a123531c8b48d6139759beab6ea1fdaeefe158 worth adding?

Reply:
Good catch. Added.